ui.tests/test-module/libs/support/commands.js

```diff
@@ -668,6 +668,13 @@ Cypress.Commands.add("expectNoConsoleErrors", () => {
     });
 });
 
+Cypress.Commands.add("getPreviewIframeBody", () => {
```

Comment:
It should be `getContentFrameBody` , in some places we have named wrongly. Let's correct from it from  now.